Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -2285,6 +2286,25 @@ static inline bool isNonReplacedInline(RenderObject& renderer)
     return renderer.isInline() && !renderer.isReplacedOrInlineBlock();
 }
 
+static bool isFlexItem(RenderObject* renderer)
+{
+    return renderer && downcast<RenderBox>(*renderer).isFlexItem();
+}
+
+static bool rendererContainingBlockHasMarginTrim(const RenderBox& renderer, std::optional<MarginTrimType> marginTrimType)
+{
+    ASSERT(renderer.isFlexItem());
+    auto* containingBlock = renderer.containingBlock();
+    if (is<HTMLHtmlElement>(renderer.element()) || containingBlock->isRenderView())
```

Comment:
/* Unofficial Review */
We may need a null check for containingBlock.

Reply:
I believe this should be fixed now from changes in https://github.com/WebKit/WebKit/pull/11640